src/amo/reducers/currentView.js

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+import { CURRENT_VIEW_SET } from 'core/constants';
+
+
+export type currentViewTypes = {|
+  // TODO: Create an addonType Type that is one of type (ADDON_TYPE_EXTENSION,
+  // ADDON_TYPE_THEME, etc.)
+  addonType: string | null,
+  isExploring: boolean | void,
+  isHomePage: boolean | void,
```

Comment:
ok, thanks for the breakdown by URL. I guess `isHomePage` is necessary but only for the header to know when to use an h1 or not.